app/loading.tsx

```diff
@@ -0,0 +1,13 @@
+import { Skeleton } from "@/components/ui/skeleton"
```

Comment:
While the skeleton looks good but I still think I have a poor understanding of when the skeleton will and won't appear and have got a couple questions.

- I don't see why cross-pathogen would be a problem, those pages actually don't share the same layout so I wouldn't expect any issues with those two pages together. I was unable to make this skeleton appear at all in my testing.
- I think you're right about the map pins Elias, stuff like loading times on the queries being made and parts of the map being rendered don't appear to be covered by a skeleton like this.
- This loading skeleton is at the base of the project. Does it also apply to the about page, the landing page, etc.?

Reply:
I think specifically because it is at the base, it only applies to the highest level of navigation. If we were to recreate the same loading.js and throw it into the pathogen dir then I think we will see the expected behaviour